File: DayTwo/JavaScript/QuestionOne.js
let greet = (text_one, text_two)=>{
    
    interleaved_text = ""
    for(let count = 0; count < text_one.length; count++){

        if(count >= 2){
            interleaved_text += text_two[count]
            interleaved_text += text_one[count]
        }

        else{
            interleaved_text += text_one[count]
            interleaved_text += text_two[count]
        }

    }
    console.log(`\"${interleaved_text}\"`)       
}

    


greet("Hello", "World")

File: DayTwo/Java/QuestionOne.java
public class QuestionOne{





public static void greet(String firstLetter, String secondLetter){
    String interleaved_text = "";
    String[] letter = firstLetter.split("");
    String[] SecondLetter = secondLetter.split("");
        for(int count = 0; count < letter.length; count++){

            if(count >= 2){
                interleaved_text += SecondLetter[count];
                interleaved_text += letter[count];
            }
            else{
                interleaved_text += letter[count];
                interleaved_text += SecondLetter[count];
            }
        }

    
     System.out.println("\"" + interleaved_text + "\"");


}


public static void main(String...ugo){
    greet("Hello","World");
}

}
